packages/marketplace/src/resources/snippets.ts

```diff
@@ -167,4 +167,10 @@ export default [
     "code": ".collection-collectionEntityHeroCard-likedSongs{ display: none; }",
     "preview": "resources/assets/snippets/hide-likedSongs-card.png",
   },
+  {
+    "title": "Fix playlist and folder position",
+    "description": "Replaces the playlists section in the left side bar to align with all the other category icons for the new UI. Screenshot includes 'Playlist icons' snippet",
+    "code": "{} .main-rootlist-rootlistItem { height: 40px; } .gJFKvJ { font-weight: 700!important; } .playlist-item__img { margin-right: 16px!important; }",
```

Comment:
Is there a reason why there's an empty {} at the beginning? 
I'm also wondering if we need to map some CSS selector, since that .gJFKvJ is likely to break down the road.

Reply:
@kyrie25 ^

Reply:
The {} has to be there for some unknown reason, otherwise it won't output the expected result. I've noticed that it can't be a different symbol, but it can be like I did in the first commit.

Reply:
It acts the exact same for me with and without the beginning `{}`

Reply:
> @kyrie25 ^

It serves no purpose afaik.

Reply:
Also hashes as selectors aren't recommended, use relative selectors instead, (`>`, `+`, `~`, `nth-child`, etc.)

Reply:
> It acts the exact same for me with and without the beginning `{}`

For me it removes the height adjustment if I don't have anything in front/in the place of the `{}`. Should I just commit it without the brackets?

Reply:
> Also hashes as selectors aren't recommended, use relative selectors instead, (`>`, `+`, `~`, `nth-child`, etc.)
> 
> > I'm also wondering if we need to map some CSS selector, since that .gJFKvJ is likely to break down the road.
> 
> 6-character classes are dynamically generated and cannot be mapped unfortunately.

Is there anything else I should adjust

Reply:
> For me it removes the height adjustment if I don't have anything in front/in the place of the `{}`. Should I just commit it without the brackets?

Yes.

> Is there anything else I should adjust

LGTM.

Reply:
> > LGTM. As soon as you address the selector it should be fine.
> > Also hashes as selectors aren't recommended, use relative selectors instead, (>, +, ~, nth-child, etc.)
> 
> Could you maybe be more specific in which one you mean? Are the classes not correct? Edit: do you mean from the !importants?

Selector is what the CSS ruleset applies on, in this case I'm addressing the one Padster mentioned above - `.gJFKvJ`.
If possible, use relative selectors instead, eg. `.main-<section>-<component> > span`

Reply:
Alright, done! I hope the css is correct now.

Reply:
That selector looks better to me. Haven't tested but I trust it does the same thing?

Reply:
I think the previous selector was only for a specific folder... now it selects all folders **and** playlists, and now it actually works. Sadly I still had to keep the !importants